src/tests/gssapi/common.h

```diff
@@ -51,6 +51,9 @@ void check_k5err(krb5_context context, const char *msg, krb5_error_code code);
 /* Display an error message containing msg and exit. */
 void errout(const char *msg);
 
+/* Import cert data */
+gss_name_t import_cert(gss_buffer_desc buf);
+
```

Comment:
We shouldn't need import_cert(), instead import_name() should interpret the 'c:' prefix.